### DISCUSSION TITLE
Hack Sonoff Zbbridge-pro et Esphome - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#51] [Comments]
# Hack Sonoff Zbbridge-pro et Esphome - Haade.fr

Comment hacker facilement le Sonoff Zbbridge-pro facilement sur Esphome et Home-Assistant et en faire une télécommande Zigbee, wifi et Bluetooth

https://haade.fr/fr/blog/flasher-zbbridge-pro-esphome-home-assistant

**Comment by pazzotranquillo** [comment]
Hello, thanks for this guide. I've tried to upload the Zigbee firmware but I'm facing issues because Could not open port socket://192.168.x.xx:6638: [Errno 61] Connection refused
